### PR TITLE
Add adsb.chaos-consulting.de

### PR DESCRIPTION
**Pull request by skymaker-c2is**
Hey there,

we just came across this list and would like to be added to it.

Cheers
Lukas

### CODE REVIEW

**Comment by rickstaa** [comment]
@skymaker-c2is, sorry for the late review; busy times 😅. Great find. I just confirmed that the website operates in the community space, not the commercial one, so everything checks out.

**Review by rickstaa — APPROVED** [review]

File: readme.md [hunk 1]
```diff
@@ -99,6 +99,7 @@ A curated list of awesome [ASD-B](https://en.wikipedia.org/wiki/Automatic_Depend
 - [PlaneSpotters.net](https://www.planespotters.net) - A civil aviation database and aggregator with a large collection of aircraft photos and information.
 - [Plane.watch](https://plane.watch) - A community hosted flight tracker.
 - [www.live-military-mode-s.eu](https://www.live-military-mode-s.eu) - A community driven flight tracker that is focused at tracking military aircraft.
+- [adsb.chaos-consulting.de](https://adsb.chaos-consulting.de) - A flight/ship/radiosonde-tracker run by hobby ethusiasts. Focused on the feeding stations. Absolutely non-commercial.
```
Comment (rickstaa) [inline readme.md:102]: How about:

```suggestion
- [adsb.chaos-consulting.de](https://adsb.chaos-consulting.de) - A non-commercial tracker for flights, ships, and radiosondes managed by enthusiasts. Focuses on the contributions of individual feeding stations.
```

I wanted to include the non-commercial inside the main description and remove the absolutely. Another option I am okay with would be `A non-commercial flight/ship/radiosonde-tracker run by hobby ethusiasts. Focused on the feeding stations`.

**Review by rickstaa — CHANGES_REQUESTED** [review]

**Comment by skymaker-c2is** [comment]
Hey @rickstaa, the changes are fine! Thank you!

**Comment by rickstaa** [comment]
@skymaker-c2is excellent! Thanks for the contribution. Merged into the main branch 🚀.

**Comment by rickstaa** [comment]
Published under https://github.com/rickstaa/awesome-adsb/releases/tag/v1.1.1.

**Pull request merged**